lib/content/gittenify.js

```diff
@@ -0,0 +1,36 @@
+import catNames from 'cat-names';
+import mutationSummary from 'mutation-summary';
+import catPhoto from './catPhoto';
+import replaceUsers from './replaceUsers';
+
+export default function gittenify(store) {
+  const userMap = {};
```

Comment:
good idea, but that is out of scope of this pull request. I just want to clean up and give it some structure. You are very welcome to create a new pull request with that feature added though :smile: